# Copy the database file into the image
COPY /home/ignis/Desktop/channel/mydatabase.db /app/data/mydatabase.db

ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref bc2c8585-f014-4399-a8dd-7a9c855fd2b8::ki8rpboqkar6uux39dsjgja3a: "/home/ignis/Desktop/channel/mydatabase.db": not found


docker run --name chatApp -v /home/ignis/Desktop/channel:/app/data -it chatapp

boken@boken-dev-machine:~/chatApp$ docker run --name chatApp -v /home/ignis/Desktop/chatApp:/app -it chatapp
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/app/main": stat /app/main: no such file or directory: unknown.
ERRO[0000] error waiting for container:   


boken@boken-dev-machine:~/chatApp$ docker run --name chatApp -it chatapp
INFO[0000] Connecting to database: db.sqlite            

2023/10/20 12:06:55 /app/models/user.go:33 UNIQUE constraint failed: users.username
[0.161ms] [rows:0] INSERT INTO `users` (`username`,`password`,`name`,`last_login_at`,`phone_no`,`designation`,`avatar_url`) VALUES ("h","123","Rahbar",1697803615,"1234567890","Developer","https://mdrahbar21.github.io/avatar.png")
ERRO[0000] Error creating user: UNIQUE constraint failed: users.username 

2023/10/20 12:06:55 /app/models/user.go:33 UNIQUE constraint failed: users.username
[0.273ms] [rows:0] INSERT INTO `users` (`username`,`password`,`name`,`last_login_at`,`phone_no`,`designation`,`avatar_url`) VALUES ("kd","123","KD LAB PC",1697803615,"1234567890","Tester","https://avatars0.githubusercontent.com/u/17098281?s=460&v=4")
ERRO[0000] Error creating user: UNIQUE constraint failed: users.username 

2023/10/20 12:06:55 /app/models/user.go:33 UNIQUE constraint failed: users.username
[0.069ms] [rows:0] INSERT INTO `users` (`username`,`password`,`name`,`last_login_at`,`phone_no`,`designation`,`avatar_url`) VALUES ("guest","guest","Guest",1697803615,"1234567890","Guest","https://avatars0.githubusercontent.com/u/17098584?s=460&v=4")
ERRO[0000] Error creating user: UNIQUE constraint failed: users.username 

2023/10/20 12:06:55 /app/models/channel.go:17 UNIQUE constraint failed: channels.name
[0.088ms] [rows:0] INSERT INTO `channels` (`name`,`description`,`created_at`,`user_username`) VALUES ("general","general convo",1697803615,"h")
INFO[0000] Starting server at port 8080                 
